MinPlatformPkg/PlatformInit/PlatformInitSmm/PlatformInitStandaloneMm.c

```diff
@@ -0,0 +1,153 @@
+  @param[in] Interface  Points to the interface instance.
+  @param[in] Handle     The handle on which the interface was installed.
+
+  @retval EFI_SUCCESS   Notification handler runs successfully.
+**/
+EFI_STATUS
+EFIAPI
+MmReadyToLockEventNotify (
+  IN CONST EFI_GUID  *Protocol,
+  IN VOID            *Interface,
+  IN EFI_HANDLE      Handle
+  )
+{
+  EFI_STATUS Status;
+
+  Status = StandaloneMmReadyToLockEventNotify (Protocol, Interface, Handle);
+  return Status;
+}
+
+/**
+  Standalone MM Ready To Boot event notification handler.
+
+  @param[in] Protocol   Points to the protocol's unique identifier.
+  @param[in] Interface  Points to the interface instance.
+  @param[in] Handle     The handle on which the interface was installed.
+
+  @retval EFI_SUCCESS   Notification handler runs successfully.
+**/
+EFI_STATUS
+EFIAPI
+MmReadyToBootEventNotify (
+  IN CONST EFI_GUID  *Protocol,
+  IN VOID            *Interface,
+  IN EFI_HANDLE      Handle
+  )
+{
+  EFI_STATUS Status;
+
+  Status = StandaloneMmReadyToBootEventNotify (Protocol, Interface, Handle);
+  return Status;
+}
+
+/**
+  Standalone MM Ready To Lock event notification handler.
+
+  @param[in] Protocol   Points to the protocol's unique identifier.
+  @param[in] Interface  Points to the interface instance.
+  @param[in] Handle     The handle on which the interface was installed.
+
+  @retval EFI_SUCCESS   Notification handler runs successfully.
+**/
+EFI_STATUS
+EFIAPI
+StandaloneMmReadyToLockEventNotify (
+  IN CONST EFI_GUID  *Protocol,
+  IN VOID            *Interface,
+  IN EFI_HANDLE      Handle
+  )
+{
+  TestPointMmReadyToLockSecureMmCommunicationBuffer ();
+  return EFI_SUCCESS;
+}
+
+/**
+  Standalone MM Ready To Boot event notification handler.
+
+  @param[in] Protocol   Points to the protocol's unique identifier.
+  @param[in] Interface  Points to the interface instance.
+  @param[in] Handle     The handle on which the interface was installed.
+
+  @retval EFI_SUCCESS   Notification handler runs successfully.
+**/
+EFI_STATUS
+EFIAPI
+StandaloneMmReadyToBootEventNotify (
+  IN CONST EFI_GUID  *Protocol,
+  IN VOID            *Interface,
+  IN EFI_HANDLE      Handle
+  )
+{
+  TestPointStandaloneMmReadyToBootStandaloneMmPageProtection ();
+  return EFI_SUCCESS;
+}
+
+/**
+  Initialize  Standalone MM Platform.
+
+  @param[in] ImageHandle       Image handle of this driver.
+  @param[in] MmSystemTable     Global system service table.
+
+  @retval EFI_SUCCESS           Initialization complete.
+  @exception EFI_UNSUPPORTED       The chipset is unsupported by this driver.
+  @retval EFI_OUT_OF_RESOURCES  Do not have enough resources to initialize the driver.
+  @retval EFI_DEVICE_ERROR      Device error, driver exits abnormally.
+**/
+EFI_STATUS
+EFIAPI
+PlatformInitStandaloneMmEntryPoint (
+  IN EFI_HANDLE           ImageHandle,
+  IN EFI_MM_SYSTEM_TABLE  *MmSystemTable
+  )
+{
+  PlatformInitMmEntryPoint ();
```

Comment:
I may have missed it. Is `PlatformInitMmEntryPoint()` defined in a header file?

Reply:
It's defined in TestPointCheckLib.h.